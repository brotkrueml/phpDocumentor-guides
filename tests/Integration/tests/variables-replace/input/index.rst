.. |rest| replace:: ReStructuredText

==============
Document Title
==============

We are using |rest| here.
