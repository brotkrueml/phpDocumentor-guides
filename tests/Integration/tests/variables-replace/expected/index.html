<!DOCTYPE html>
<html lang="en">
<head>
    <title>Document Title</title>

</head>
<body>
    <div class="section" id="document-title">
    <h1>Document Title</h1>

            <p>We are using ReStructuredText here.</p>
    </div>

</body>
</html>
